shell/platform/darwin/ios/framework/Headers/FlutterEngineGroup.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+
+#import "FlutterEngine.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Represents a collection of FlutterEngines who share resources which allows
+ * them to be created with less time const and occupy less memory than just
+ * creating multiple FlutterEngines.
+ *
+ * @see http://flutter.dev/go/multiple-engines
```

Comment:
Yep, good point.  I think this is helpful now in case someone looks at it before the docs are written.  You think we should have nothing or just create an issue for updating them once the documentation is written?

Reply:
Let's let it be a code comment so it doesn't accidentally become part of the final guides. While we're here, we should also caveat the whole file on top and say this is a work in process and shouldn't be used.

Reply:
Addressed in xiao feedback 4 - Added comment to the class